### ISSUE TITLE
Restoring from checkpoint failed

### ISSUE THREAD

**Issue by CQSamBoom** [endernewton/tf-faster-rcnn#481]
**I have trained a new Faster RCNN using my own training set. It' ok during training and testing.
but I using my trained mode to run demo.py for my picture, there are:**

 Restoring from checkpoint failed. This is most likely due to a mismatch between the current graph and the graph from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Assign requires shapes of both tensors to match. lhs shape= [2048,84] rhs shape= [2048,24]

who can help me ,please.

**Comment by rnsandeep** [comment]
check on the number of anchors you have used during train and test. Both should match.

**Comment by CQSamBoom** [comment]
> check on the number of anchors you have used during train and test. Both should match.

 I find the code about anchors and find out the question , I forget correct n-class , think you all the same

**Comment by rnsandeep** [comment]
Yes by default pascal-voc has 21 classes. so 21*4. you could have changed it to 6 so 6*4.

**Comment by devendraswamy** [comment]
> > check on the number of anchors you have used during train and test. Both should match.
> 
> I find the code about anchors and find out the question , I forget correct n-class , think you all the same

Thank you , i am also getting same error ,so please help me where i need to modify ??

Thanking you .

**Comment by devendraswamy** [comment]
> Yes by default pascal-voc has 21 classes. so 21x4. you could have changed it to 6 so 6x4.

 I am also getting same error ,so please help me where i need to modify ??

Thanking you .

**Comment by DK223** [comment]
I have the same question. Could you tell me where you find the code about anchors? I can't find it.

**Comment by DK223** [comment]
>

> > check on the number of anchors you have used during train and test. Both should match.
> 
> I find the code about anchors and find out the question , I forget correct n-class , think you all the same

I have the same question. Could you tell me where you find the code about anchors? I can't find it.
